cypress/integration/errorPage.js

```diff
@@ -0,0 +1,31 @@
+import { getElement, renderedTitle } from '../support/bodyTestHelper';
+import { testResponseCode } from '../support/metaTestHelper';
+
+describe('Article Body Tests', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    cy.visit(`/news/articles/c9rpqyepmypo`, { failOnStatusCode: false });
```

Comment:
It would be good to pull in the asset using `config.assets` as you have done for the other integration tests (https://github.com/bbc/simorgh/blob/1da766ddded195bab157a36914d968cc9cfd9cb1/cypress/integration/newsBodySpec.js#L19).  
To do this, you'll likely want to create a new variable such as `nonexistent` for this non-existent article id.